Pod/Classes/Math/FloatingPoint+Scale.swift

```diff
@@ -13,11 +13,12 @@ public extension FloatingPoint {
     /// - Parameters:
     ///   - source: The range to interpret the number as being a part of.
     ///   - destination: The range to map the number to.
+    ///   - timingTransform: An optional transform mapping input percentage to output percentage. Defaults to `nil`
```

Comment:
Fix name of parameter as well. I miss `valid_docs`. Also, could do `curve` or `function` instead of `transform`.